package.json

```diff
@@ -34,6 +34,7 @@
     "@babel/preset-react": "7.26.3",
     "@babel/preset-typescript": "7.26.0",
     "@patternfly/react-icons": "5.4.2",
+    "@playwright/test": "^1.50.1",
```

Comment:
Can you please pin the versions of newly added deps?

Reply:
oh woops, yes of course